src/actions/guides/database/database_setup.cr

```diff
@@ -15,20 +15,61 @@ class Guides::Database::DatabaseSetup < GuideAction
 
     > If you're using the postgres app on macOS, be sure to install [CLI tools](https://postgresapp.com/documentation/cli-tools.html).
 
+    ### Avram Configuration
+
+    Avram comes with an `Avram::Credentials` class for configuring your database credentials. This helps to catch incorrect settings at compile-time
+    like malformatted usernames and such.
+
+    To configure Lucky to connect to your database, open up your `config/database.cr` file. You'll find a few options within the `AppDatabase` configure block.
+
     ### Standard Options
 
-    To configure Lucky to connect to your database, open up your `config/database.cr` file.
-    You'll find a few standard options within the `AppDatabase` configure block.
+    Set the `settings.credentials` to `Avram::Credentials.new` and pass in these options:
+
+    * database : `String` - This is the name of your database. The default is set at the top of this file.
+    * hostname : `String?` - The host where your database is located. Generally "localhost".
+    * username : `String?` - Your database user.
+    * password : `String?` - Your password.
+    * port : `Int32?` - The port to connect to. Default is `5432`
+    * query : `String?` - A query string of connection pool settings.
+
+    ```crystal
+    # config/database.cr
+    AppDatabase.configure do |settings|
+      settings.credentials = Avram::Credentials.new(
+        database: database_name,
+        hostname: "localhost",
+        port: 5432,
+        username: "postgres",
+        query: "initial_pool_size=5&retry_attempts=2"
+      )
+    end
+    ```
+
+    On most systems, you can leave the password blank if your setup doesn't require a password. If you wish to use password-less connections for local development,
+    and leaving the password blank doesn't work, please see [Installing Postgres](#{Guides::GettingStarted::Installing.path(anchor: Guides::GettingStarted::Installing::ANCHOR_POSTGRESQL)})
+    for more tips.
+
+    ### Using Postgres connection string
+
+    Avram also supports using a standard connection string whether you want to use unix sockets or connect with credentials. For this, we can just set
+    an environment variable like `DATABASE_URL` and parse it with `Avram::Credentials.parse`
```

Comment:
Might be worth mentioning that this is already in config/database.cr. The way it is phrased now makes it seem like they would need to do this themselves.